MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -3605,10 +3605,10 @@
 				CLANG_ENABLE_OBJC_WEAK = YES;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 107;
+				CURRENT_PROJECT_VERSION = 108;
 				DEFINES_MODULE = YES;
 				DYLIB_COMPATIBILITY_VERSION = 1;
-				DYLIB_CURRENT_VERSION = 107;
+				DYLIB_CURRENT_VERSION = 108;
```

Comment:
`108` is already taken by `2.6.0-alpha.2`. Should we update it to `109`?

Reply:
Yep, makes total sense.